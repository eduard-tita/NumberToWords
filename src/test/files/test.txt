0
-3
17
109
245
5,002
234,019
-1,000,013
coffee
1000000000000000000000000000000000000000000000000000000
