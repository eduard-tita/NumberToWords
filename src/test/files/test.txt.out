0 : Zero
-3 : Minus three
17 : Seventeen
109 : One hundred and nine
245 : Two hundred and forty five
5,002 : Five thousand and two
234,019 : Two hundred and thirty four thousand and nineteen
-1,000,013 : Minus one million and thirteen
coffee : Invalid number
1000000000000000000000000000000000000000000000000000000 : Number too large
